### ISSUE TITLE
Little Computer People - fails to run on U64, v1.33 3.8

### ISSUE THREAD

**Issue by xlar54** [GideonZ/1541ultimate#152]
The screen which asks your name and date/time is garbled and past that, the game will freeze.  There is a cracked version on csdb.dk which does run, but its a .prg file.  My guess is that most have some kind of timing issue to the drive that even at 1mhz fails.  No idea why the multicolor bitmap screen would be garbled.  Im curious myself and may take a look, but i need to first pull down the latest codebase and understand it.
[Little Computer People - Casey.zip](https://github.com/GideonZ/1541ultimate/files/4790158/Little.Computer.People.-.Casey.zip)

**Comment by Jumpman64** [comment]
@xlar54 My fb posting was rubbish, it just took longer than expected to load the game.
I tried your D64 and it ran fine, the research book screen is correct. The TAT, SCA and original version is running too. Please check the attached working original version on your machine.
[little_test.zip](https://github.com/GideonZ/1541ultimate/files/4794084/little_test.zip)

only "casey version" shows a black pixel on the right side in the blue sky, the others are ok.

Tested on U64 rev1.2 V1.33 - 3.8

**Comment by xlar54** [comment]
Thanks for checking before bothering Gideon with this one.  I could have a setting somewhere that is breaking it.  Lets close this issue.

**Comment by xlar54** [comment]
Here's a pic of what I get, using your disk, on v1.28 (3.7)  (so its not due to turbo modes).  Has to be something in my configuration.

![20200620_213313](https://user-images.githubusercontent.com/1909642/85215433-16cb7c00-b33e-11ea-90f4-9fbeb69ede26.jpg)

**Comment by xlar54** [comment]
Aha.  So I went back and reflashed the board and it told me that the RTC had failed.  Makes sense because I tried setting the clock and it would not retain my setting.  I have another board that flashed the same way without issue, and LCP is working just fine.  I asked Gideon if this could just be a bad battery.  Might try to replace it and see if any difference.  But this may explain to others, should they face similar symptoms.
